Fix: Logs could be exported via ConsoleExporter after LoggerFactory is disposed #1848 (#3578)
open-telemetry/opentelemetry-dotnet: src/OpenTelemetry.Exporter.Console/CHANGELOG.md

## Unreleased

* Changed the behavior of `ConsoleExporter`, the exporter will stop outputting
  the data if it is disposed.

>   ([#1848](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1848))

Reviewer: ```suggestion
  ([#3578](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3578))
```

## Unreleased

> * Changed error handling, `ConsoleExporter` will only output an error message

Reviewer: The description doesn't seem correct to me.

Author: Update it, please help to check my English

> * Changed error handling, `ConsoleExporter` will output error messages,
>   tell user where dispose happened and drop the data
>   if `Export` is invoked after the exporter is disposed.
>   ([#1848](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1848))

Reviewer: ```suggestion
* Changed the behavior of `ConsoleExporter`, the exporter will stop outputting the data if it is disposed.
  ([#1848](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1848))
```

Author: Thank you so much, updated
